test/fw/ptl/utils/plugins/ptl_report_json.py

```diff
@@ -118,7 +119,8 @@ def get_json(self, data, prev_data=None):
             'end_time': str(data['end_time']),
             'measurements': []
         }
-        tcshort['requirements'] = data['requirements']
+        tcshort['requirements'] = getattr(data, 'requirements',
```

Comment:
We shouldn't be doing this. PTLJsonData.get_json() is just a converter of test data passed to it in dictionary format into json format. It should not be creating or populating data in this method.